server.js

```diff
@@ -19,6 +19,32 @@ const helpers = require('./utils/helpers');
 const app = express();
 const PORT = process.env.PORT || 3001;
 
+passport.use(
+  new FacebookStrategy(
+    {
+      // clientID: process.env.FACEBOOK_APP_ID,
+      clientID: 'YOUR_FACEBOOK_APP_ID', //added this
+      clientSecret: process.env.FACEBOOK_APP_SECRET,
+      callbackURL: process.env.FACEBOOK_CALLBACK_URL,
```

Comment:
I think it should be like this?

```suggestion
      clientID: process.env.FACEBOOK_CLIENT_ID
      clientSecret: process.env.FACEBOOK_CLIENT_SECRET,
      callbackURL: process.env.FACEBOOK_CALLBACK_URL,
```

Reply:
go for it!